pkg/get/tools.go

```diff
@@ -237,5 +237,20 @@ https://github.com/inlets/inletsctl/releases/download/{{.Version}}/{{$fileName}}
 `,
 		})
 
+	tools = append(tools,
+		Tool{
+			Owner:   "kubernetes-sigs",
+			Repo:    "kubebuilder",
+			Name:    "kubebuilder",
+			Version: "2.3.1",
+			URLTemplate: `
```

Comment:
Please add support for the arm64 binary, and add a unit test for the three combos

Reply:
@alexellis all the required changes were added